lib/util/stream_utils.js

```diff
@@ -108,7 +108,8 @@ shaka.util.StreamUtils = class {
         const currentCodec =
           MimeUtils.getNormalizedCodec(stream.codecs);
         if (previousCodec == currentCodec) {
-          if (stream.bandwidth > previousStream.bandwidth) {
+          if (!stream.bandwidth ||
```

Comment:
Now that I think about it, when the bandwidth is unknown it's `undefined`, right? And not `null` or whatever.

That means we should also check for `!previousStream.bandwidth`, since even `0 > undefined == false`.

Reply:
Done!